packages/xod-fs/test/fixtures/workspace/__lib__/xod/core/pot/patch.cpp

```diff
@@ -0,0 +1 @@
+// should be imported into impl.cpp
```

Comment:
What’s this?

Reply:
Oops. It was an outdated placeholder. Fixed!